requirements/runtime.txt

```diff
@@ -1,6 +1,7 @@
 faiss-gpu==1.6.1
 future
 matplotlib
+mmcls
```

Comment:
Should we also inform users to install mmcls in install.md?

Reply:
yes，also I will add mmcls to requirements/mminstall.txt